AirCasting/Services/DownloadMeasurmentsService.swift

```diff
@@ -67,6 +67,10 @@ final class DownloadMeasurementsService: MeasurementUpdatingService {
                     } catch {
                         assertionFailure("Failed to save context \(error)")
                     }
+                    PersistenceController.shared.performBackgroundTask { bgContext in
```

Comment:
There is already a reference to `persistenceController` in this class. Why use a shared instance?

Reply:
`DownloadMeasurementsService` could be defiantly easier to setup if ever need if `RemoveOldMeasurementsService ` would be constructed as a lazy var.